J-MaFf/RocketNavigator
# testing

- Tests on the pi
- Add files via upload
- Apply Black formatting
- Tried copy pasting instead of replacing the files to preserve commit history.
- Removed testing folder.
- Need address.
- Apply Black formatting
- test2
- Apply Black formatting
- test3
- Apply Black formatting
- Change to self.i2c
- temp accel and baro working
- Apply Black formatting
- Removed sensorValue var assignment.
- Updated view to look prettier.
- Apply Black formatting
- sensors return lists not tuples
- Apply Black formatting
- changes
- Apply Black formatting
- Changing view.
- Apply Black formatting
- added barometer
- Apply Black formatting
- update_view method in controler now prints the sensor name.
- Fixed typo
- Sensor name is no longer included in the data array. Instead, it is printed in the view.
- Apply Black formatting
- changed veiw a bit
- Apply Black formatting
- added self
- made output look good
- Apply Black formatting
- donezo
